core/src/main/java/org/testcontainers/containers/image/DockerJavaImageData.java

```diff
@@ -0,0 +1,54 @@
+package org.testcontainers.containers.image;
+
+import com.github.dockerjava.api.command.InspectImageResponse;
+import java.time.OffsetDateTime;
+import java.util.List;
+import lombok.Data;
+import lombok.Value;
+
+@Data
+@Value
+public class DockerJavaImageData implements ImageData {
+
+    private InspectImageResponse inspectImageResponse;
```

Comment:
```suggestion
    InspectImageResponse inspectImageResponse;
```
`@Value`@ makes it `private final` by default